src/main/java/core/basesyntax/controller/ConsolHendler.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.RequestBarvinokDao;
+import core.basesyntax.dao.RequestBarvinokDaoImpl;
+import core.basesyntax.dao.RequestVeselkaDao;
+import core.basesyntax.dao.RequestVeselkaDaoImpl;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.RequestBarvinok;
+import core.basesyntax.model.RequestVeselka;
+
+import java.util.Scanner;
+
+public class ConsolHendler {
+    @Inject
+    public static RequestVeselkaDao requestVeselkaDao;
+
+    @Inject
+    public static RequestBarvinokDao requestBarvinokDao;
+
+    public static void handle1() {
```

Comment:
let's create more understandable name of the method

Reply:
ok